### PR TITLE
Everything for the about us page

### PR DESCRIPTION
**Pull request by finben**
A few notes.

1. I decided on the web view for the link. I just feel like it still looks better than just opening the url externally. But lmk what you think.

2. I could not get the "here" link to be in line with the text so long as it had a tap action on it. So... I went with just making the whole line the link secretly. Personally, I don't think anyone would ever click there unless they meant to link to it anyways. But if you can find a better way lmk. I left a comment in the code about it. Hopefully, they support this later. 

3. I made two tweaks to the figma wording.  I made it "this study Bible" instead of "the study Bible". To me this just sounds right since we where just talking about this study Bible. But if you or Emily don't agree we can change it back to what you guys had. 
The other thing is adding a period after Col. since it is an abbreviation. Again if you guys don't like it we can change it back to the spec

### CODE REVIEW

**Comment by finben** [comment]
Simulator Screen Shot - iPhone 8 - 2020-06-11 at 23 00 42
Simulator Screen Shot - iPhone 8 - 2020-06-11 at 23 00 46

File: Hymns.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -157,6 +157,9 @@
 		B56305B9244223D7003BC132 /* FontViewModifier.swift in Sources */ = {isa = PBXBuildFile; fileRef = B56305B8244223D7003BC132 /* FontViewModifier.swift */; };
 		B5809DFA244F60FD00EF0C63 /* FavoriteEntity.swift in Sources */ = {isa = PBXBuildFile; fileRef = B5809DF9244F60FD00EF0C63 /* FavoriteEntity.swift */; };
 		B5809DFC244F614000EF0C63 /* FavoritesViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = B5809DFB244F614000EF0C63 /* FavoritesViewModel.swift */; };
+		B582E4452493266700329DA5 /* AbousUsInfoView.swift in Sources */ = {isa = PBXBuildFile; fileRef = B582E4442493266700329DA5 /* AbousUsInfoView.swift */; };
```
Comment (skywalkerdude) [inline Hymns.xcodeproj/project.pbxproj:160]: Aboutsus? 😛

File: Hymns/Settings/AbousUsInfoView.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+import UIKit
+
+struct AbousUsInfoView: View {
```
Comment (skywalkerdude) [inline Hymns/Settings/AbousUsInfoView.swift:4]: Abousus? 😛

File: Hymns/Settings/AbousUsInfoView.swift [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    @State var webView = false
+    @Environment(\.presentationMode) var presentationMode
+
+    var body: some View {
+        VStack {
+            Group<AnyView> {
+                if webView {
+                    return WebView(url: URL(string: "https://online.recoveryversion.bible/")!).eraseToAnyView()
```
Comment (skywalkerdude) [inline Hymns/Settings/AbousUsInfoView.swift:12]: I think it should be bfa, not online recovery version

File: Hymns/Settings/AbousUsInfoView.swift [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+                        }.padding().padding(.top).foregroundColor(.primary)
+                        HStack {
+                            Text("Hello There 👋").font(.largeTitle).fontWeight(.bold)
+                            Spacer()
+                        }.padding()
+                        Text("We are a group of Christians who love Jesus Christ. We created this app as a free resource to help people easily access hymns. Many of these hymns are inspired by verses from the Recovery Version Study Bible.").padding()
+                        Text("Let the word of Christ dwell in you richly in all wisdom, teaching and admonishing one another with psalms and hymns and spiritual songs, singing with grace in your hearts to God.").padding()
+                        HStack {
+                            Spacer()
+                            Text("- Col. 3:16").font(.callout).fontWeight(.bold).padding()
```
Comment (skywalkerdude) [inline Hymns/Settings/AbousUsInfoView.swift:32]: I also made some tweaks to the wording as well. We can talk about this in the issue w.r.t exactly what it should say and what it should link to

File: Hymns/Settings/AbousUsInfoView.swift [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+                        }
+                        //Cant get this dumb thing in line. It won't concatonate the text since it has a tapGesture. Right now the whole sentence will link you. //https://stackoverflow.com/questions/59359730/is-it-possible-to-add-an-in-line-button-within-a-text
+                        Group {
+                            Text("To learn more about this study Bible visit the site") + Text(" here.").fontWeight(.bold).underline()
+                        }
```
Comment (skywalkerdude) [inline Hymns/Settings/AbousUsInfoView.swift:37]: That's fine. The entire thing can be a link too, and it wouldn't be a probem.

**Comment by skywalkerdude** [comment]
closes https://github.com/HymnalDreamTeam/Hymns-iOS/issues/198

**Pull request merged**